### ISSUE TITLE
Screen reader - implementation - (current)

### ISSUE THREAD

**Issue by Magical1** [wp-bootstrap/wp-bootstrap-navwalker#343]
Is there an easy way of appending their to the end of the active link item.

```
<li class="nav-item active">
        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
</li> 
```

Many thanks in advance.

**Comment by pattonwebz** [comment]
Hey @Magical1,

I'll take a look at how best to implement this into the walker and see what I can come up with before the weekend :)

This does seem like a valuable addition for accessibility reasons so I asked some of the people that are involved with WP a11y team for their thoughts too so that I can be sure I get this added and done right first time.

Thanks for highlighting this and requesting the feature :smile:

**Comment by pattonwebz** [comment]
I had a quick chat with some a11y team members and they suggested looking at perhaps the `aria-current` attribute for announcing the current page to screen readers instead of hidden screen reader text - but also suggesting that in many cases such an announce may not be required or duplicated so having it enabled all the time by default might not be best way to handle it.

I am not an accessibility expert so I don't know for certain of the ideal way to implement this.

I'm leaving this open for a while to decide best move forward, if anyone has an opinion or links to resources about this sharing those would be appreciated.

**Comment by LouisMilotte** [comment]
create a js bound to document load that finds .active under your primary navigation that seeks out the anchor child and appends the necessary HTML. A bit messy, but unless the walker is changed to do this automatically  - really the best way I found out to do it.

**Comment by IanDelMar** [comment]
This was added by #434

**Comment by IanDelMar** [comment]
@pattonwebz The issue has been closed in #434
